.gitignore

```diff
@@ -4,8 +4,8 @@ out/
 target/
 build/
 *.iml/
-.idea/
```

Comment:
Why remove .idea ?

Reply:
It is listed twice in this file, if you open the whole file you will see the `.idea/` entry at the very top of it.

Reply:
Ah I didn't realize that.